src/rapidcore.redis.csproj

```diff
@@ -1,22 +1,30 @@
-<Project Sdk="Microsoft.NET.Sdk">
-
-  <PropertyGroup>
-    <TargetFramework>netstandard1.6</TargetFramework>
-    <RootNamespace>RapidCore</RootNamespace>
-    <Version>0.0.0</Version>
-    <Title>RapidCore</Title>
-    <Authors>nover</Authors>
-    <Description>RapidCore.Redis is redis dependend code for the RapidCore library</Description>
-    <PackageTags>dotnet core;core;rapidcore;business;business logic;help;helpers;library;redis;lock;locking;distributed locking;</PackageTags>
-    <PackageProjectUrl>https://github.com/rapidcore/rapidcore.redis/</PackageProjectUrl>
-    <PackageLicenseUrl>https://raw.githubusercontent.com/rapidcore/rapidcore.redis/master/LICENSE</PackageLicenseUrl>
-    <PackageRequireLicenseAcceptance>false</PackageRequireLicenseAcceptance>
-    <RepositoryType>git</RepositoryType>
-    <RepositoryUrl>https://github.com/rapidcore/rapidcore.redis/</RepositoryUrl>
-    <PackageId>RapidCore.Redis</PackageId>
-    <Company>SKARP ApS</Company>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
-    <CodeAnalysisRuleSet>../stylecop.ruleset</CodeAnalysisRuleSet>
-  </PropertyGroup>
-
-</Project>
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.6</TargetFramework>
+    <RootNamespace>RapidCore</RootNamespace>
```

Comment:
I see your point, however that makes it slightly less obvious that it comes from the RapidCore.Redis package.